Scripts/package.py

```diff
@@ -122,25 +122,30 @@ def build(self):
 
         if path.exists(self.buildDirectory):
             shutil.rmtree(self.buildDirectory)
+        env = dict(os.environ)
+        env["RUNTIME_IDENTIFIER"] = self.target
```

Comment:
Is it not possible to update `os.environ` so you don't need the dictionary copying and additional parameters?

Reply:
Looks like this new `env` argument is not used in all the calls to `run`, only in the three publishing directives. I think it makes sense to control the environment modification to affect only the child processes who need it. It'll also be easier to reason about than changing the environment globally.